Microsoft.Toolkit.Mvvm/Microsoft.Toolkit.Mvvm.csproj

```diff
@@ -4,16 +4,16 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <LangVersion>8.0</LangVersion>
     <Nullable>enable</Nullable>
-    <Title>Windows Community Toolkit Mvvm .NET Standard</Title>
+    <Title>Windows Community Toolkit MVVM .NET Standard</Title>
```

Comment:
```suggestion
    <Title>Windows Community Toolkit MVVM Toolkit</Title>
```

Reply:
Question, should this just be "Windows Community MVVM Toolkit", to avoid the "Toolkit MVVM Toolkit" repetition? 🤔

Reply:
You'd think, but they're both proper names... we could maybe add a dash between them or something to delineate?

Reply:
Mmh not sure about the dash, wouldn't want people to start referencing to this package with both versions of the name, MVVM Toolkit and MVVM-Toolkit, that might be confusing. I'll just leave it as is for now then! 😊